runtime.md

```diff
@@ -55,18 +55,24 @@ The lifecycle describes the timeline of events that happen from when a container
     If the runtime is unable to create the environment specified in the [`config.json`](config.md), it MUST [generate an error](#errors).
     While the resources requested in the [`config.json`](config.md) MUST be created, the user-specified program (from [`process`](config.md#process)) MUST NOT be run at this time.
     Any updates to [`config.json`](config.md) after this step MUST NOT affect the container.
-3. Runtime's [`start`](runtime.md#start) command is invoked with the unique identifier of the container.
-4. The [prestart hooks](config.md#prestart) MUST be invoked by the runtime.
-    If any prestart hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 9.
-5. The runtime MUST run the user-specified program, as specified by [`process`](config.md#process).
-6. The [poststart hooks](config.md#poststart) MUST be invoked by the runtime.
-    If any poststart hook fails, the runtime MUST [log a warning](#warnings), but the remaining hooks and lifecycle continue as if the hook had succeeded.
-7. The container process exits.
+3. The [`prestart` hooks](config.md#prestart) MUST be invoked by the runtime.
+    If any `prestart` hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 12.
+4. The [`create-runtime` hooks](config.md#create-runtime-hooks) MUST be invoked by the runtime.
+    If any `create-runtime` hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 12.
+5. The [`create-container` hooks](config.md#create-container-hooks) MUST be invoked by the runtime.
+    If any `create-container` hook fails, the runtime MUST [generate an error](#errors), stop the container, and continue the lifecycle at step 12.
+6. Runtime's [`start`](runtime.md#start) command is invoked with the unique identifier of the container.
```

Comment:
Since `pivot_root` features prominently in the specification of the `create-*` hooks, I think it should be listed in this life-cycle presentation as well.

Reply:
Will do!

Reply:
I realized this is something we probably don't want to specify. In case runtimes don't execute `pivot_root` as part of the `create` command, we are better off not indicating when runtimes should execute specific operations (or the equivalent).

Reply:
@h-vetinari I'm keeping this conversation open, If you have a specific idea it's probably worth discussing !

Reply:
It's OK if `pivot_root` can not be reflected accurately in this very linear lifecycle description (as I naively thought); feel free to resolve this comment.